core-services/prow/02_config/openshift/cluster-authentication-operator/_pluginconfig.yaml

```diff
@@ -1,14 +1,24 @@
 label:
   restricted_labels:
     openshift/cluster-authentication-operator:
-    - allowed_users:
-      - stlaz
-      label: backport-risk-assessed
     - allowed_users:
       - xingxingxia
       - yaoli-redhat
       - y4sht
       label: cherry-pick-approved
+    - allowed_users:
+      - Miciah
+      - danehans
+      - deads2k
+      - frobware
+      - ironcladlou
+      - knobunc
+      - pravisankar
+      - ramr
+      - sgreene570
+      - stlaz
+      - sttts
```

Comment:
IMO, this list seems too big, I'd probably pick only top-level approvers, but I'll let auth team decide.